### PR TITLE
Update SKILL.tsv

### PR DESCRIPTION
**Pull request by Sunaries**
Minor fixes. 
- **SKILL_20150317_001796** (https://forum.treeofsavior.com/t/a-thread-for-any-issues-i-find/29884/8)
- **SKILL_20150317_001915** (https://forum.treeofsavior.com/t/bonfire-buff-has-a-extra-increase-at-the-end-of-the-description/34229)
- **SKILL_20150317_002051, SKILL_20150317_002068, SKILL_20150317_002069, SKILL_20150317_002070** (https://forum.treeofsavior.com/t/swash-buckling-attribute-typo/33537)

### CODE REVIEW

File: SKILL.tsv [hunk 1]
```diff
@@ -2065,10 +2065,10 @@ SKILL_20150317_002064	Restrain: Duration
 SKILL_20150317_002065	Duration of [Retrain] will increase by 5 seconds per level
 SKILL_20150317_002066	Thrust: Chance of Critical
 SKILL_20150317_002067	Critical probability of [Thrust] is increased by 10% per attribute level.
-SKILL_20150317_002068	Swoosh Buckling: Max. HP
-SKILL_20150317_002069	Max. HP of character increases by 5% per attribute level while [Swoosh Buckling] is enabled.
-SKILL_20150317_002070	Swoosh Buckling: Range
-SKILL_20150317_002071	Provocation range of [Swoosh Buckling] increase by 0.2m per level.
+SKILL_20150317_002068	Swash Buckling: Maximum HP
+SKILL_20150317_002069	Increases the maximum HP of your character by 5% per attribute level while [Swoosh Buckling] is active.
```
Comment (Ihysoal) [inline SKILL.tsv]: [Swoosh Buckling]

Comment (Sunaries) [inline SKILL.tsv]: Thanks for your attentiveness. :+1: 
Fixed.

**Comment by Ihysoal** [comment]
Also please read these issues: https://github.com/Treeofsavior/EnglishTranslation/issues/731 https://github.com/Treeofsavior/EnglishTranslation/issues/672

They describe and discuss standards for spell and attribute discriptions, you could use them and/or chip in your opinion!

**Comment by imcgames** [comment]
The Pull Request is closed for issues regarding copyrights as stated in our main page. 
We will be updating you with details in the near future regarding this matter so please hold your Pull Requests until things the issue is resolved. 
We apologize for the inconvenience caused.

**Pull request closed without merging**